bib/cmd/bootc-image-builder/main.go

```diff
@@ -260,17 +276,26 @@ func main() {
 		Long:                  rootCmd.Long,
 		Args:                  cobra.ExactArgs(1),
 		DisableFlagsInUseLine: true,
-		Run:                   build,
+		Run:                   cmdBuild,
 	}
 	rootCmd.AddCommand(buildCmd)
+	manifestCmd := &cobra.Command{
+		Use:                   "manifest",
+		Long:                  rootCmd.Long,
+		Args:                  cobra.ExactArgs(1),
+		DisableFlagsInUseLine: true,
+		Run:                   cmdManifest,
+	}
+	rootCmd.AddCommand(manifestCmd)
+	manifestCmd.Flags().String("rpmmd", "/var/cache/osbuild/rpmmd", "rpm metadata cache directory")
+	manifestCmd.Flags().String("config", "", "build config file")
+	manifestCmd.Flags().String("type", "qcow2", "image type to build [qcow2, ami]")
+	manifestCmd.Flags().Bool("tls-verify", true, "require HTTPS and verify certificates when contacting registries")
```

Comment:
We also need an outputdir for the manifest, don't we?

Reply:
Happy to do this if you prefer it this way, so far the manifest is just output to stdout which is fine for my use-case but if `--output` is desired for manifest I'm happy to add it.